lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -147,6 +147,10 @@ void LocalEnforcer::setup(
   }
 }
 
+void LocalEnforcer::PollStats() {
+  pipelined_client_->poll_stats(0, 0, NULL);
```

Comment:
add some type of a default callback it can be simple for now
```
 pipelined_client_->poll_stats(0, 0, [](Status status, RuleRecordTable resp) {
        if (!status.ok()) {
          MLOG(MERROR) << "Coudn't poll stats " << ": " << status.error_message();
        }
      });
```